int32 robot_id
int32 task_id
float64 bid_value
